pkg/services/publicdashboards/public_dashboard_service_mock.go

```diff
@@ -50,7 +50,7 @@ func (_m *FakePublicDashboardService) Create(ctx context.Context, u *user.Signed
 }
 
 // Delete provides a mock function with given fields: ctx, uid
-func (_m *FakePublicDashboardService) Delete(ctx context.Context, uid string) error {
+func (_m *FakePublicDashboardService) Delete(ctx context.Context, uid string, dashboardUid string) error {
```

Comment:
It is necessary to change some logic inside this function, not only adding the new parameter in the signature, please use the code I shared in another comment.

Reply:
done, works